webapp/lib/controller.dart

```diff
@@ -176,6 +169,18 @@ loadProjectConfigurationView() {
   view.contentView.renderView(new view.ProjectConfigurationView(model.projectConfigurationFormData, model.additionalProjectConfigurationLanguages));
 }
 
+void addPackage(String packageName) {
+  model.packageConfigurationData[packageName] = new model.Configuration()
+    ..availableTags = model.tags
+    ..hasAllTags = {}
+    ..containsLastInTurnTags = {}
+    ..hasNoneTags = {}
+    ..suggestedReplies = []
+    ..addsTags = {};
```

Comment:
After the changes to `model.Configuration`:
```suggestion
    ..availableTags = model.tags;
```